artview/components/select_region.py

```diff
@@ -13,7 +14,7 @@
 
 from ..core import (Variable, Component, common, VariableChoose,
                     componentsList, QtGui, QtCore)
-from ..core.points import Points, write_points_csv, read_points_csv
+from ..core.points import Points
 
```

Comment:
Will it be possible to start a PointsDisplay from SelectRegion? I think we should consider this.

I hate to lose the functionality. Will this be added before we finalize a first round of manual_unfold?